fastly/kafka_test.go

```diff
@@ -38,7 +38,7 @@ func TestClient_Kafkas(t *testing.T) {
 			TLSClientCert:    clientCert,
 			TLSClientKey:     clientKey,
 			ParseLogKeyvals:  true,
-			RequestMaxBytes:  1024,
+			RequestMaxBytes:  2048,
```

Comment:
One minor nit. Seeing as this number is repeated 3 times in this file and an API enforced limit, it feels like it should be factored into a constant value at the top of the file. Just my style preference.

Reply:
I'm usually a "more than twice, refactor" kinda guy. So I can dig it 👍🏻